os/sdk-modifying-coreos.md

```diff
@@ -168,7 +168,19 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
-If you encounter errors with KVM, verify that it is supported by your CPU and enabled in the BIOS.
+If you encounter errors with KVM, verify that virtualization is supported by your CPU by doing `egrep '(vmx|svm)' /proc/cpuinfo`. The `/dev/kvm` directory will be in your host OS when virtualization is enabled in the BIOS.
+
+The `./coreos_production_qemu.sh` file can be found in the `~/trunk/src/build/images/amd64-usr/latest` directory of the booted filesystem.
+
+#### Several Ways to Boot:
+
+After `image_to_vm.sh` compeletes, do `./coreos_production_qemu.sh -curses` to launch a graphical interface to login to the Container Linux vm.
```

Comment:
s/compeletes/completes/

Reply:
Okay, thanks! Good catch... too used to autocorrect 😅